scripts/format.sh

```diff
@@ -10,6 +10,10 @@ set -o errexit -o pipefail -o nounset
 
 readonly CLANG_FORMAT_VERSION=11.0.0
 
+if [[ $(uname -i) == 'aarch64' ]]; then
+  echo 'Formatting is disabled on arm for the time being'
+  exit
+fi
```

Comment:
we cannot get a version of clang format for arm that is locked to the same version as the linux/mac binaries we get from mason. we should stop using mason but because we are in this retarded mode of having to have all formatting look identical, and because the linter is software and goes through revisions, if we dont have matchign revisions across platforms we cant get the same output. this situation kind of makes you lose hope for humanity

Reply:
Is there an alternative to mason? Would love to get rid of that too, being unmaintained etc. Conan & vcpkg are only doing libs, no exes (as projects). 

FWIW, I’m very much pro formatter on a project with uhh say heterogeneous styles😄python was an eyesore to me before I got it linted & formatted.

Reply:
i know im alone in my heterodox opinion however you have to admit the irony that you want to enforce conformity but cant because the thing that does it is different on different platforms and changes randomly over time. its like saying lets adopt a standard because people are doing this way too many different ways and then when you implement the standard it allows for you to implement it in different ways. the only thing you gained was the pain of trying to adhere to the standard the core problem is unsolved.

as far as a real solution goes we could run clang_format via a docker image that locks the version. i think i proposed that before. i know its a bit clunky but it is robust to all the problems. it does mean that docker is a required dependency to lint the code which is a bit nuts but short of building clang format from source its the only real option that or some other magical formatter